Source: backported from http://git.savannah.gnu.org/cgit/libtool.git/patch/?id=f9adc3ec9c0ba05664e6f17efc0c86dddcdfe1b1
Upstream: in git
Reason: bash 4 prints a warning when the specified locale is invalid,
        breaking the test suite's diff

--- libtool-2.2.6/tests/localization.at~	2009-07-15 11:48:47.000000000 +0100
+++ libtool-2.2.6/tests/localization.at	2009-07-15 11:49:09.000000000 +0100
@@ -47,6 +47,9 @@
 mv -f stderr expected-stderr
 AT_CHECK([$LIBTOOL --mode=compile --tag=CC $CC $CPPFLAGS $CFLAGS -c a.c || exit 1],
 	 [1], [stdout], [stderr])
+# Avoid this error:
+#   /bin/sh: warning: setlocale: LC_ALL: cannot change locale (de_DE)
+AT_CHECK([grep LC_ALL stdout stderr && exit 77], [1])
 AT_CHECK([diff expected-stderr stderr])
 LTBASE=`$ECHO "$LIBTOOL" | sed 's,^.*/,,'`
 AT_CHECK([grep -v "^$LTBASE: compile" stdout | diff expected-stdout -])
